releasenotes/notes/holland-vevn-9eb62140a2c8ff56.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - Holland now runs inside a python virtual environment
+    by default. By default, the virtual environment will
+    be placed in /openstack/venvs/holland-release-tag.
```

Comment:
sorry, I know it seems obvious, but you probably want to change this to ```holland-<release tag>``` to be very clear